Manager/UserManager.php

```diff
@@ -61,7 +82,7 @@ public function getUserById($userId)
      */
     public function hasRole(UserInterface $user, $role)
     {
-        return in_array(strtoupper($role), $user->getRoles(), true);
+        return $this->authorizationChecker->isGranted($role);
```

Comment:
Resources/views/Ticket/show.html.twig
```
{% if message.userObject|isTicketAdmin('ROLE_TICKET_ADMIN') %}<span
   class="label label-danger">{{ 'LABEL_ADMIN'|trans }}</span> {% endif %}
```

This checks the user of the message to see if they are admin.  I.e. not always the logged in user.  

![screen shot 2016-10-11 at 08 26 56](https://cloud.githubusercontent.com/assets/1474182/19262000/84cecb3c-8f8c-11e6-847e-9641914ed7d2.png)

I can't think of any way around this, other than storing this information in the db.